frontend/context/notification.tsx

```diff
@@ -10,7 +10,7 @@ type InitialStateType = {
   notification: INotification | null;
   renderFlash: (
     alertType: "success" | "error" | "warning-filled" | null,
-    message: string | null,
+    message: JSX.Element | string | null,
```

Comment:
Updates flash message to allow JSX.Elements because Figma mock has bolded text in it.